apps/agreement/test/agreement/agreement_new_action.js

```diff
@@ -216,7 +216,7 @@ contract('Agreement', ([_, owner, submitter, someone]) => {
                               const sentEth = bn(1)
```

Comment:
Should we also remove the `value` parameter in [`itHandlesNewActionsCorrectly()`](https://github.com/aragon/aragon-apps/blob/forward_non_payable/apps/agreement/test/agreement/agreement_new_action.js#L66)?